README.md

```diff
@@ -0,0 +1,107 @@
+# Canvas-image-cover [![Build Status](https://secure.travis-ci.org/noblesamurai/canvas-image-cover.png?branch=master)](http://travis-ci.org/noblesamurai/canvas-image-cover) [![NPM version](https://badge-me.herokuapp.com/api/npm/canvas-image-cover.png)](http://badges.enytc.com/for/npm/canvas-image-cover)
+
+> Cover a canvas with an image given a locked centre point and zoom factor.
+
+## Usage
+
+```js
+const cover = require('canvas-cover-image');
+const Canvas = require('canvas');
+const Image = Canvas.Image;
+const canvas = new Canvas(200, 200);
+const ctx = canvas.getContext('2d');
+const img;
+
+// Assume image is 200x200px
+fs.readFile(__dirname + '/images/squid.png', function(err, squid){
+  if (err) throw err;
+  img = new Image;
+  img.src = squid;
+  overlay(img);
+  zoom(img);
+  offset(img);
+});
+
+// Perfect overlay of image to canvas
+function overlay (img) {
+  cover(ctx, img, 0, 0, 200, 200);
+}
+
+// Overlay image (at 2X zoom) to canvas.
+function zoom (img) {
+  cover(ctx, img, 0, 0, 200, 200, { zoom: 2 });
+}
+
+// Overlay image, and zoom with 0, 0 locked as centre point of zoom.
+function offset(img) {
+  cover(ctx, img, 0, 0, 200, 200, { zoom: 2, cx: 0, cy: 0 });
+}
+```
+
+## API
+<a name="cover"></a>
+
+## cover(ctx, img, x, y, width, height)
+Draw an image on the canvas at a given centre point.  The image is zoomed
+(with locked aspect ratio) so that it covers the canvas along both axis.
+If zoom is greater than 1, the image is then zoomed by the the given zoom factor,
+with centre point locked as per the arguments given.
+
+**Kind**: global function
+
+| Param | Type | Description |
+| --- | --- | --- |
+| ctx | <code>CanvasRenderingContext2D</code> | Canvas context to render to |
+| img | <code>Canvas.Image</code> \| <code>Canvas</code> | Image to render |
+| x | <code>number</code> |  |
+| y | <code>number</code> |  |
+| width | <code>number</code> |  |
+| height | <code>number</code> |  |
+| opts.cx | <code>number</code> | Gradient center x position defined as a value between   0 left side of the box to 1 right side of the box (defaults to 0.5). |
+| opts.cy | <code>number</code> | Gradient center y position defined as a value between   0 and 1 (defaults to 0.5). |
+| opts.zoom | <code>number</code> | Image scaling factor (default 1 - cover only). |
+| opts.alpha | <code>number</code> | Alpha value between 0 transparent and 1 opaque. |
+
+Note: To regenerate this section from the jsdoc run `npm run docs` and paste
+the output above.
+
+## Installation
```

Comment:
I've been meaning to query having this at the bottom in our template? I kind of think this should be at the top... given you usually need to install before you use?